### PR TITLE
Kenny Completed

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by darkness198** [comment]
Reviewed 1 on 1

File: Answers.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+1. Preprocessing is basically taking a complicated language and creating a simpler way of writing code and having a program basically translate a more human readable material to the standard format that a computer can read.
```
Comment (frankfaustino) [inline Answers.md:1]: Preprocressing adds extra functionality to CSS that's found in other programming languages: _variables_, _operations_ and _functions_ to name a few.

File: Answers.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+2. npm install -g less
+3. rotate
+4. display: flexbox;
```
Comment (frankfaustino) [inline Answers.md:4]: 4. `xs`, `sm`, `md`, `lg`, `xl`
5. `.d-flex`

File: less/layout.less [hunk 3]
```diff
@@ -5,4 +5,13 @@ html, body {
 h1, h2, h3, h4, h5 {
     font-family: 'Russo One', sans-serif;
     color: @secondary-color;
+}
+.container{
+    max-width: 900px;
+}
+nav {
+    background-color: #eeeeee;
```
Comment (frankfaustino) [inline less/layout.less:13]: We can use `@main-color` here in the LESS file since our colors are already picked out in `variables.less`

**Pull request closed without merging**